jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/JibBuildRunnerTest.java

```diff
@@ -250,5 +250,6 @@ public void testBuildImage_writesImageJson() throws Exception {
     Assert.assertEquals(imageId, metadataOutput.getImageId());
     Assert.assertEquals(digest, metadataOutput.getImageDigest());
     Assert.assertEquals(tags, ImmutableSet.copyOf(metadataOutput.getTags()));
+    Assert.assertTrue(metadataOutput.isImagePushed());
```

Comment:
Looks like we're missing a mockito setup above. How about we add this in line 246: `Mockito.when(mockJibContainer.isImagePushed()).thenReturn(true);`

Reply:
Aaagh! Missed out commiting that line :-) Fixed, sorry about that